fuels-core/src/code_gen/abigen.rs

```diff
@@ -464,8 +503,8 @@ mod tests {
             assert!(contract.custom_structs.contains_key(name));
         }
 
-        let bindings = contract.generate().unwrap();
-        bindings.write(std::io::stdout()).unwrap();
+        let bindings = contract.generate();
+        assert!(matches!(bindings, Err(Error::MissingData(_))));
```

Comment:
Do you mind adding a comment here explaining why we're expecting to get this error? The name of the test itself doesn't mention that, the reader would be confused. Would be nice to explain that we're just testing multiple inputs generations at once and that would exceed the limit (4).